lib/webrtc/rtcpeerconnection/chrome.js

```diff
@@ -24,128 +23,126 @@ var isUnifiedPlan = sdpUtils.getSdpFormat() === 'unified';
+      },
+      _tracksToSSRCs: {
+        value: new Map(),
+        writable: true
+      },
+      localDescription: {
+        enumerable: true,
+        get: function() {
+          return this._pendingLocalOffer ? this._pendingLocalOffer : peerConnection.localDescription;
+        }
+      },
+      remoteDescription: {
+        enumerable: true,
+        get: function() {
+          return this._pendingRemoteOffer ? this._pendingRemoteOffer : peerConnection.remoteDescription;
+        }
+      },
+      signalingState: {
+        enumerable: true,
+        get: function() {
+          if (this._pendingLocalOffer) {
+            return 'have-local-offer';
+          } else if (this._pendingRemoteOffer) {
+            return 'have-remote-offer';
+          }
+          return peerConnection.signalingState;
         }
-        return peerConnection.signalingState;
       }
-    }
-  });
+    });
 
-  var self = this;
+    let self = this;
 
-  peerConnection.addEventListener('datachannel', function ondatachannel(event) {
-    shimDataChannel(event.channel);
-    self.dispatchEvent(event);
-  });
+    peerConnection.addEventListener('datachannel', function ondatachannel(event) {
+      shimDataChannel(event.channel);
+      self.dispatchEvent(event);
+    });
 
-  peerConnection.addEventListener('signalingstatechange', function onsignalingstatechange() {
-    if (peerConnection.signalingState === 'stable') {
-      self._appliedTracksToSSRCs = new Map(self._tracksToSSRCs);
-    }
-    if (!self._pendingLocalOffer && !self._pendingRemoteOffer) {
-      self.dispatchEvent.apply(self, arguments);
-    }
-  });
 
-  peerConnection.ontrack = function ontrack() {
-    // NOTE(mroberts): adapter.js's "track" event shim only kicks off if we set
-    // the ontrack property of the RTCPeerConnection.
-  };
+    peerConnection.addEventListener('signalingstatechange', function onsignalingstatechange() {
+      if (peerConnection.signalingState === 'stable') {
+        self._appliedTracksToSSRCs = new Map(self._tracksToSSRCs);
+      }
+      if (!self._pendingLocalOffer && !self._pendingRemoteOffer) {
+        self.dispatchEvent.apply(self, arguments);
+      }
+    });
 
-  if (typeof RTCPeerConnection.prototype.addTrack !== 'function') {
-    peerConnection.addStream(this._localStream);
-  }
-  util.proxyProperties(RTCPeerConnection.prototype, this, peerConnection);
-}
+    peerConnection.ontrack = function ontrack() {
+      // NOTE(mroberts): adapter.js's "track" event shim only kicks off if we set
+      // the ontrack property of the RTCPeerConnection.
+    };
 
-inherits(ChromeRTCPeerConnection, EventTarget);
+    if (typeof RTCPeerConnection.prototype.addTrack !== 'function') {
+      peerConnection.addStream(this._localStream);
+    }
+    util.proxyProperties(RTCPeerConnection.prototype, this, peerConnection);
+  }
 
-if (typeof RTCPeerConnection.prototype.addTrack !== 'function') {
+  // if (typeof RTCPeerConnection.addTrack !== 'function') {
```

Comment:
This may be causing the test failures. You need to shim `addTrack()` only if the method is not available on `RTCPeerConnection`.